test/necb/unit_tests/tests/test_necb_skylights.rb

```diff
@@ -33,13 +33,17 @@ def test_necb_skylights()
     # to toplight occupied spaces below.
     @buildings = [
       'FullServiceRestaurant',
-      # 'LargeOffice',
-      # 'MediumOffice',
+      'LargeOffice',
+      'MediumOffice',
       # 'NorthernEducation',
-      # 'QuickServiceRestaurant',
+      'QuickServiceRestaurant',
       # 'SmallOffice'
     ]
 
+    # NOTE: Skipping NorthernEducation for now:
+    #   - Standards.Model.rb:5432: warning: instance variable @space_type_map not initialized
+    #   - model works in isolation (e.g. SDK 3.6.1, 3.7.0, 3.8.0)
```

Comment:
The _NorthernEducation_ model works with _addSkylights_ just fine in isolation. This is likely more of a BTAP prototype family thing. Setting aside for now.